lib/tasks/detailed_guides.rake

```diff
@@ -0,0 +1,32 @@
+namespace :detailed_guides do
+  desc "Relocate detailed guides referenced in Topics"
+  task :relocate_to_guidance => :environment do
+    panopticon = CollectionsPublisher.services(:panopticon)
+
+    page = 1
+    while true
```

Comment:
This pagination stuff should be unnecessary.  There's already pagination support built into gds-api-adapters (See [`ListResponse#with_subsequent_pages`](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/list_response.rb#L79))